src/gate/library/liberty/translate.cpp

```diff
@@ -388,7 +388,7 @@ bool translateModuleToGNet(
 void translateDesignToGNet(
     const RTlil::Design *des,
     NetData &vec) {
-  for (auto [IdString, Module]: des->modules_) {
+  for (auto &[IdString, Module]: des->modules_) {
```

Comment:
utopia-eda/src/gate/library/liberty/translate.cpp:391:8: warning: 'auto &' can be declared as 'const auto &' [llvm-qualified-auto]